File: Assets/_makeYOURlevel_LITE/Standard Assets/Character Controllers/Sources/Scripts/Bound.cs
using UnityEngine;
using System.Collections;

public class Bound : MonoBehaviour {
	void Start () {
		MeshFilter[] meshes = FindObjectsOfType (typeof(MeshFilter)) as MeshFilter[];
		for (int i=0;i<meshes.Length;i++)
	    {
			  if(meshes[i].mesh){
				MeshHelper.Subdivide(meshes[i].mesh,4);
				meshes[i].mesh.bounds.Expand(150000);

			  }
		}
	}

	void Update () {

}

}

File: Assets/BigBoxBound/Bound.js
#pragma strict

function Start () {
    for (var t: MeshFilter in GameObject.FindObjectsOfType(MeshFilter))
    {
	  var mesh : Mesh = t.mesh;
	  if(mesh){
	  	mesh.bounds.extents = Vector3(15000,15000,15000); 
	  }
	}
}

function Update () {

}